CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+# 3.0.0-rc.4 (2018-12-21)
+
+- Fixed datepicker and timepicker components to properly transform date/time object on initial load in reactive forms [#7](https://github.com/blackbaud/skyux-datetime/issues/7).
```

Comment:
I'm wondering if we should say 
`properly transform initial values on initial load`

I say this as it may not always be an object (such as a date or time string) when it initially loads. The way I read what you have would make me think that the change only affects things if an object is loaded.